diff --git a/node_modules/@fantox01/lyrics-scraper/dist/Driver/request.js b/node_modules/@fantox01/lyrics-scraper/dist/Driver/request.js
index fc04142..42b1356 100644
--- a/node_modules/@fantox01/lyrics-scraper/dist/Driver/request.js
+++ b/node_modules/@fantox01/lyrics-scraper/dist/Driver/request.js
@@ -49,7 +49,7 @@ var fetchData = function (searchTerm) {
             switch (_g.label) {
                 case 0:
                     _g.trys.push([0, 2, , 3]);
-                    target = "https://genius.com/api/search/multi?per_page=1&q=" + searchTerm;
+                    target = "https://cors-anywhere.herokuapp.com/https://genius.com/api/search/multi?per_page=1&q=" + searchTerm;
                     return [4 /*yield*/, axios_1.default.get(target)];
                 case 1:
                     res = _g.sent();
